python/mqt/syrec/__init__.py

```
@@ -14,16 +14,16 @@
 import sys
+from pathlib import Path
 
 # under Windows, make sure to add the appropriate DLL directory to the PATH
 if sys.platform == "win32":
 
     def _dll_patch() -> None:
         """Add the DLL directory to the PATH."""
-        import os
-        import sysconfig
-        from pathlib import Path
+        import sysconfig  # noqa: PLC0415
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)